pepper-apis/studybuilder-cli/studies/osteo/patches/about-child-v2.conf

```diff
@@ -0,0 +1,74 @@
+{
+  "searchSectionByQuestionCode":"WHO_IS_FILLING_ABOUTYOU",
+  "shownExpr":"""false""",
+  "renderMode": "CHECKBOX_LIST",
+  "who_filling_q_child": {
+    "question": {
+      include required("../../snippets/picklist-question-single-list.conf"),
+      "renderMode": "CHECKBOX_LIST",
+      "hideNumber": true,
+      "stableId": "WHO_IS_FILLING_ABOUTYOU_CHILD",
+      "promptTemplate": {
+        "templateType": "HTML",
+        "templateText": "$WHO_IS_FILLING_ABOUTYOU_CHILD_prompt",
+        "variables": [
+          {
+            "name": "WHO_IS_FILLING_ABOUTYOU_CHILD_prompt",
+            "translations": [
+              {
+                "language": "en",
+                "text": "Who is filling out this survey?"
+              }
+            ]
+          }
+        ]
+      },
+      "picklistOptions": [
+        {
+          "stableId": "TOGETHER_WITH_CHILD",
+          "optionLabelTemplate": {
+            "templateType": "TEXT",
+            "templateText": "$WHO_IS_FILLING_together_with_child",
+            "variables": [
+              {
+                "name": "WHO_IS_FILLING_together_with_child",
+                "translations": [
+                  {
+                    "language": "en",
+                    "text": "My child has been diagnosed with osteosarcoma, and we are filling out this survey together."
+                  }
+                ]
+              }
+            ]
+          },
+          "detailLabelTemplate": null,
+          "allowDetails": false,
+          "exclusive": false
+        },
+        {
+          "stableId": "ON_BEHALF_OF_CHILD",
+          "optionLabelTemplate": {
+            "templateType": "TEXT",
+            "templateText": "$WHO_IS_FILLING_on_behalf_of_child",
+            "variables": [
+              {
+                "name": "WHO_IS_FILLING_on_behalf_of_child",
+                "translations": [
+                  {
+                    "language": "en",
+                    "text": "My child has been diagnosed with osteosarcoma, and I am filling out this survey on their behalf."
+                  }
+                ]
+              }
+            ]
+          },
+          "detailLabelTemplate": null,
+          "allowDetails": false,
+          "exclusive": false
```

Comment:
Now that we are hiding this option for adult, please return these to radio buttons for child consent so that only one can be selected (not sure if that means setting `exclusive` as well--you know better than I do).